Core/Graphics/Source/Unix/GraphicsPlatform.cpp

```diff
@@ -1,15 +1,14 @@
-#include <Babylon/Graphics.h>
 #include <Babylon/GraphicsPlatform.h>
-#include "../GraphicsImpl.h"
+#include <GraphicsImpl.h>
 
-constexpr float MILLIMETERS_TO_INCHES = 0.03937;
+constexpr float MILLIMETERS_TO_INCHES{0.03937f};
 
 namespace Babylon
 {
     void GraphicsImpl::ConfigureBgfxPlatformData(const GraphicsConfiguration& config, bgfx::PlatformData& pd)
     {
         pd.ndt = nullptr;
-        pd.nwh = (void*)config.WindowPtr;
+        pd.nwh = static_cast<void*>(config.WindowPtr);
```

Comment:
As you mentioned before, we're still using `void*` for our linux window type, is there any clever casting that we can do in the shared code to allow for us to use an `unsigned long` "pointer"

Reply:
_Hmm_, interesting... I'm not sure if this PR is the right place to change that, but this brings up a good point. Where actually is the shared code that would have to deal with this cast? In other words, if you change `WindowType` to be `unsigned long` and change this line to cast from that to the `void*` that bgfx is expecting, where does that break? I can check myself when I'm back on Monday and can fire up my Linux VM to try it, was just curious if you happened to know off the top of your head.

Reply:
This seems only relevant when we're casting to/from void* when setting the native window handle that we provide to bgfx:

https://github.com/BabylonJS/BabylonNative/blob/8371b79805f2647dc7c43aa1f3e0a5552f23e2f6/Core/Graphics/Source/GraphicsImpl.cpp#L45

Reply:
I don't think this minor detail should hold up the rest of these changes, however.